docker/fedora-25/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM        fedora:25
 MAINTAINER  Anton Povarov "anton.povarov@gmail.com"
 
+# TODO: Use multi-stage builds https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
 RUN         dnf install -y git-core gcc gcc-c++ boost-devel cmake autoconf automake libtool mariadb-devel
 RUN         dnf install -y file hostname mariadb-server
 
+ADD         . /_src/pinba2
+
 RUN         mkdir -p /_src && cd /_src && \
-			git clone https://github.com/anton-povarov/pinba2 && \
-			git clone https://github.com/anton-povarov/meow && \
-			git clone https://github.com/nanomsg/nanomsg ;
+			git clone --branch master --single-branch --depth 1  https://github.com/anton-povarov/meow && \
```

Comment:
Unfortunately, https://github.com/anton-povarov/meow/releases has no releases so we have to use `master` which is no the best option.